common4j/src/test/com/microsoft/identity/common/java/telemetry/TelemetryTest.java

```diff
@@ -158,6 +158,21 @@ public void onReceived(List<Map<String, String>> telemetryData) {
         Telemetry.getInstance().flush();
     }
 
+
+    @Test
+    public void testTelemetryMap() {
+        List<Map<String,String>> telemetryData = Telemetry.getInstance().getMap();
+        final Map<String, String> map = telemetryData.get(0);
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.App.NAME));
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.App.BUILD));
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.Device.MODEL));
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.Device.NAME));
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.Os.NAME));
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.Os.VERSION));
+        Assert.assertTrue(map.containsKey(TelemetryEventStrings.Device.TIMEZONE));
+    }
```

Comment:
Can you also add an Assert here to confirm that the data is still present internally? Just call getMap again and check that the data is still there (essentially that it isn't flushed)